package.json

```diff
@@ -203,6 +202,7 @@
     "update-notifier": "^2.5.0",
     "uuid": "^2.0.3",
     "write-file-atomic": "^2.4.3",
-    "yaml-ast-parser": "0.0.43"
+    "yaml-ast-parser": "0.0.43",
```

Comment:
Why is that dependency added?

Reply:
Sorry, ignore that comment. I didn't properly read, that it's just comma that's added